pycondor/command_line.py

```diff
@@ -0,0 +1,138 @@
+
+from __future__ import division, print_function
+import os
+import sys
+import time
+from collections import namedtuple
+import argparse
+
+_states = ['Done', 'Pre', 'Queued', 'Post', 'Ready', 'UnReady', 'Failed']
+Status = namedtuple('Status', _states)
+
+
+def _parse_lines(lines):
+    '''Function to parse input list of lines from dagman out file
+
+    Parameters
+    ----------
+    lines : list
+        Iterable of lines to parse.
+
+    Returns
+    -------
+    status : Status
+        Status namedtuple that contains the current number of jobs that are
+        done, queued, ready, failed, etc. If no line is found indicating the
+        current dagman status, an empty Status object is returned.
+    '''
+    node_counts = []
+    # Reverse order of lines to get the most recent status of the dagman
+    lines = lines[::-1]
+    for idx, line in enumerate(lines):
+        if 'Done     Pre   Queued    Post   Ready   Un-Ready   Failed' in line:
+            num_line = lines[idx-2]
+            node_counts = [i for i in num_line.split(' ')
+                           if i != '' and ':' not in i and '/' not in i]
+            node_counts = map(int, node_counts)
+            break
+
+    if node_counts:
+        status = Status(*node_counts)
+    else:
+        status = Status(*[0]*len(_states))
+
+    return status
+
+
+def status_genreator(dag_out_file):
```

Comment:
Did you mean `status_generator`?

Reply:
That's definitely what I mean. Good catch @kreczko, thanks!